pkg/pgclient/client.go

```diff
@@ -65,15 +70,12 @@ type Client struct {
 // NewClient creates a new PostgreSQL client
 func NewClient(cfg *Config, readHist prometheus.ObserverVec) (*Client, error) {
 	connectionStr := cfg.GetConnectionStr()
-
-	maxProcs := runtime.GOMAXPROCS(-1)
-	if maxProcs <= 0 {
-		maxProcs = runtime.NumCPU()
-	}
-	if maxProcs <= 0 {
-		maxProcs = 1
+	minConnections, maxConnections, numCopiers, err := cfg.GetNumConnections()
+	if err != nil {
+		log.Error("err configuring number of connections", util.MaskPassword(err.Error()))
+		return nil, err
 	}
-	connectionPool, err := pgxpool.Connect(context.Background(), connectionStr+fmt.Sprintf(" pool_max_conns=%d pool_min_conns=%d", maxProcs*pgmodel.ConnectionsPerProc, maxProcs))
+	connectionPool, err := pgxpool.Connect(context.Background(), connectionStr+fmt.Sprintf(" pool_max_conns=%d pool_min_conns=%d", maxConnections, minConnections))
```

Comment:
I would add a log-line with the minConnections, maxConnections and numCopiers here. Would ease debugging on client machines

Reply:
line 80 should have the min and max at least
```
log.Info("msg", util.MaskPassword(connectionStr))
```